src/scrapers/main.py

```diff
@@ -1,16 +1,19 @@
 import logging
 from optparse import OptionParser
 
+from models import InitiativeGroup
 from platformen import NLvoorElkaar, WijAmsterdam, CoronaHelpersScraper
-    # HeldNodig, MensenDieWillenHelpen, Zorgheldenauto, PuurPapendrecht, NijmegenOost
+# HeldNodig, MensenDieWillenHelpen, Zorgheldenauto, PuurPapendrecht, NijmegenOost
 from tools import Geocoder
 
 logging.basicConfig(level=logging.DEBUG)
 
 parser = OptionParser(add_help_option=False)
```

Comment:
`parser = ArgumentParser()`

Reply:
De `add_option`'s worden dan `add_argument`, volgens mij is de rest drop-in compatible